sdk/python/foundation-models/system/content-safety/deploy_with_content_safety.ipynb

```diff
@@ -0,0 +1,803 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# How to create an Safety-Enabled LLaMa Online Endpoint\n",
+    "## This notebook will walk you through the steps to create a Safety-Enabled LLaMa Online Endpoint.\n",
+    "### The steps are:\n",
+    "1. Create a Content Safety for moderating the request from user and response from the LLaMa Online Endpoint\n",
+    "2. Create a new LLaMa Online Endpoint\n",
+    "3. Create a new Safety-Enabled LLaMa Online Endpoint with a custom score.py file which will be used to moderate the request and response"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 1. Prerequisites\n",
+    "#### 1.1 Install Dependencies"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%pip install azure-identity==1.14.0b1\n",
```

Comment:
Can the notebook be run using non-prerelease and non beta versions?

Reply:
fixed